### ISSUE TITLE
when batchsize is small ,the lr should small too?

### ISSUE THREAD

**Issue by York1996OutLook** [sshaoshuai/PointRCNN#157]
in your paper ,state 2 trainging batchsize is 256,but most of us do not have so many gpu.for examle ,at most ,i use batchsize 24,
Should I reduce the learning rate because it is very important to the training process？

**Comment by wangshichang-726** [comment]
yes,you should do this
